eth/tracers/internal/tracers/assets.go

```diff
@@ -46,9 +45,8 @@ func bindataRead(data []byte, name string) ([]byte, error) {
 }
 
 type asset struct {
-	bytes  []byte
-	info   os.FileInfo
-	digest [sha256.Size]byte
+	bytes []byte
+	info  os.FileInfo
```

Comment:
this doesn't seem to be using the same go-bindata as we've used previously. You don't have to fix that though, it's better if one of us regenerates it before merging.